packages/core/medusa-test-utils/src/medusa-test-runner.ts

```diff
@@ -172,6 +163,16 @@ export function medusaIntegrationTestRunner({
       throw error
     }
 
+    try {
+      console.log(`Running migrations and syncing links ${dbName}`)
+      dbUtils.pgConnection_ = await initDb({
+        env,
+      })
```

Comment:
q: before this, the db doesn't exist? Doesn't the bootstrapApp function need a pg connection to run the defaults workflow?

Reply:
Ohh yeah. Actually, have to split this into two parts.

- Init DB before the app starts.
- Run migrations and sync links after the app starts. Because, we need these operations to pick migrations and links from the app too